nuitka/build/SconsUtils.py

```diff
@@ -605,7 +605,7 @@ def getMsvcVersionString(env):
 
 
 def getMsvcVersion(env):
-    value = getMsvcVersionString(env)
+    value = getMsvcVersionString(env) or "14.38.33130"
```

Comment:
That's not an option, MSVC is not mandatory.

Reply:
Oh.. Sorry, I forgot to remove it XD